test/e2e_node/cpumanager_test.go

```diff
@@ -166,6 +167,14 @@ var _ = SIGDescribe("CPU Manager", ginkgo.Ordered, framework.WithSerial(), featu
 		podMap = make(map[string]*v1.Pod)
 	})
 
+	ginkgo.JustBeforeEach(func(ctx context.Context) {
+		// the framework is initialized using an injected BeforeEach node, so the
+		// earliest we can do is to initialize the other objects here
+		localNode = getLocalNode(ctx, f)           // bootstrap
+		cpuDetails = cpuDetailsFromNode(localNode) // bootstrap
```

Comment:
Let's consolidate this just like we have done in other places:
```suggestion
		cpuDetails = cpuDetailsFromNode(getLocalNode(ctx, f)) // bootstrap
```

Reply:
we will still to update `localNode` in some places. I'll still see if I can refactor this further